tmp/lanelet2_extension/lib/utilities.cpp

```diff
@@ -536,19 +536,18 @@ bool lineStringToPolygon(
   const lanelet::ConstLineString3d & linestring, lanelet::ConstPolygon3d * polygon)
 {
   if (polygon == nullptr) {
-    RCLCPP_ERROR_STREAM(
```

Comment:
I think we can keep this as an ERROR since this never happens unless there is a bug in the code.

Reply:
Got it and done in https://github.com/autowarefoundation/autoware_common/pull/181/commits/b00adb9f01039ea803733712ccd8cd8624b92b33